source/modes/GameMode.cpp

```diff
@@ -489,7 +519,18 @@ bool GameMode::mousePressed(const OIS::MouseEvent& arg, OIS::MouseButtonID id)
     if (id == OIS::MB_Right)
     {
         inputManager.mRMouseDown = true;
-
+        inputManager.mRStartDragX = inputManager.mXPos;
+        inputManager.mRStartDragY = inputManager.mYPos;
+        if(arg.state.Y.abs < 0.1*arg.state.height || arg.state.Y.abs > 0.9*arg.state.height)
+        {
+            inputManager.mDragType=InputManager::rotateAxisX;
+            return true;
+        }
+        else if(arg.state.X.abs > 0.9*arg.state.width || arg.state.X.abs < 0.1*arg.state.width)
```

Comment:
"you have to clic on the GUI to move the camera. And if you clic on the GUI, you won't get here"
Ahh alright the bottom case ....